specifyweb/specify/migration_utils/update_schema_config.py

```
@@ -245,32 +243,29 @@ def bulk_create_splocaleitemstr_idempotent(Splocaleitemstr, rows: list[dict]) ->
 
     return total_created
 
+class TableDefaults(TypedDict):
+    name: NotRequired[str]
+    desc: NotRequired[str]
+    items: "NotRequired[dict[str, FieldDefaults]]"
+
 def update_table_schema_config_with_defaults(
     table_name,
     discipline_id: int,
     description: str = None,
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)